lang/tags/data_null.py

```diff
@@ -4,13 +4,18 @@
 mod = Module()
 
 mod.tag("code_data_null", desc="Tag for enabling commands relating to null")
+mod.list("code_data_null", desc="null-like value (e.g. Python `None`, C++ `nullptr`)")
+
+ctx.lists["self.code_data_null"] = {
```

Comment:
Looks like yiu dropped "no". Was that intentional?

Reply:
I'm actually surprised `state no` was a command for null in the first place. It's inconsistent with the `is not (null|none)` command.

I figure this space could be useful for someone that might want to use `state yes/no` for booleans instead.

Reply:
My guess is that was there for recognition accuracy, as "null" can be challenging for talon to recognise. I wouldn't say I'm strictly opposed to removing it